ddp-workspace/projects/ddp-sdk/src/lib/services/activity/validators/activityUniqueValidationRule.ts

```diff
@@ -0,0 +1,40 @@
+import { ActivityQuestionBlock } from '../../../models/activity/activityQuestionBlock';
+import { ActivityAbstractValidationRule } from './activityAbstractValidationRule';
+import { QuestionType } from '../../../models/activity/questionType';
+import { ActivityCompositeQuestionBlock } from 'ddp-sdk';
+
+export class ActivityUniqueValidationRule extends ActivityAbstractValidationRule {
+    constructor(question: ActivityQuestionBlock<any>) {
+        super(question);
+    }
+
+    public recalculate(): boolean {
+        const {answer: answers} = this.question;
```

Comment:
Can we add the type for `answers`. Pretty important detail in this chunk of code. I get very confused looking at two dimensional arrays. Please make it super-obvious what I am looking at (in the sample code I provided in comment below I named variables and added types because I (myself) was having a hard time tracking what was what in my own snippet).

Reply:
done!